tensorflow/contrib/rnn/python/ops/core_rnn_cell_impl.py

```diff
@@ -100,11 +102,11 @@ def output_size(self):
 
   def call(self, inputs, state):
     """Gated recurrent unit (GRU) with nunits cells."""
-    with vs.variable_scope("gates"):  # Reset gate and update gate.
+    with vs.variable_scope("gates", initializer=self._initializer):  # Reset gate and update gate.
```

Comment:
@ebrevdo are you concerned that eventually someone might want to pass different initializers to these different variables?